mkosi

```diff
@@ -1028,7 +1028,7 @@ def install_opensuse(args, workspace, run_build_script):
     #
     # Install the "minimal" package set.
     #
-    subprocess.run(cmdline + ["-t", "pattern", "minimal_base"], check=True)
+    subprocess.run(cmdline + ["patterns-base-minimal_base"], check=True)
```

Comment:
It looks like  we lost the visibility flag. I reported this issue and it should be fixed soon.

So this change isn't correct IMHO and you should have reported this issue to the openSUSE community instead.

Reply:
This has been fixed now.

Reply:
Great, thanks.

Reply:
I continuing to need this patch for both leap and tumbleweed

Reply:
@juaningan 

For TW I don't see this problem:
```
> zypper se -s -t pattern minimal
Loading repository data...
Reading installed packages...

S | Name         | Type    | Version       | Arch   | Repository             
--+--------------+---------+---------------+--------+------------------------
i | minimal_base | pattern | 20170319-15.1 | x86_64 | openSUSE-Tumbleweed-Oss
v | minimal_base | pattern | 20170319-15.1 | i586   | openSUSE-Tumbleweed-Oss
```
The package in Virtualization/mkosi still has this patch in btw because at the time v3 has been released that was not fixed.

And is Leap supported by mkosi?

Reply:
Same error for both releases:

```
# mkosi -d opensuse -r tumbleweed
# mkosi -d opensuse -r 42.3
```
```
(...)
Building repository 'Main' cache .....................................................................................................................................................................[done]
Building repository 'Updates' cache ..................................................................................................................................................................[done]
Loading repository data...                                                                            
Reading installed packages...                      
'minimal_base' not found in package names. Trying capabilities.                                                                                                                                             No provider of 'minimal_base' found.
‣ Unmounting Package Cache...                      
‣ Unmounting Package Cache complete.               
‣ Unmounting image...                              
‣ Unmounting image complete.                       
‣ Detaching image file...                          
‣ Detaching image file complete.                   
Traceback (most recent call last):                 
  File "/home/juanjop/mkosi/mkosi", line 3390, in <module>                                            
    main()                                         
  File "/home/juanjop/mkosi/mkosi", line 3380, in main                                                
    build_stuff(args)                              
  File "/home/juanjop/mkosi/mkosi", line 3284, in build_stuff                                         
    raw, tar, root_hash = build_image(args, workspace, run_build_script=False)                        
  File "/home/juanjop/mkosi/mkosi", line 3133, in build_image                                         
    install_distribution(args, workspace.name, run_build_script, cached)                              
  File "/home/juanjop/mkosi/mkosi", line 1442, in install_distribution                                
    install[args.distribution](args, workspace, run_build_script)                                     
  File "/usr/lib/python3.6/contextlib.py", line 52, in inner                                          
    return func(*args, **kwds)                     
  File "/home/juanjop/mkosi/mkosi", line 1381, in install_opensuse                                    
    run(cmdline + ["-t", "pattern", "minimal_base"], check=True)                                      
  File "/usr/lib/python3.6/subprocess.py", line 418, in run                                           
    output=stdout, stderr=stderr)                  
subprocess.CalledProcessError: Command '['zypper', '--root', '/var/tmp/mkosi-9ww544b1/root', '--gpg-auto-import-keys', 'install', '-y', '--no-recommends', '-t', 'pattern', 'minimal_base']' returned non-zero exit status 104.
```